stable/postgresql/templates/secrets.yaml

```diff
@@ -14,6 +15,4 @@ data:
   {{ else }}
   postgres-password: {{ randAlphaNum 10 | b64enc | quote }}
   {{ end }}
-  {{- if .Values.metrics.customMetrics }}
```

Comment:
doing so breaks BC, so this should be a MAJOR version bump. At least if this chart follows semver 🙂

Reply:
Is there some docs on what exactly entails a semver change? I did a helm upgrade from the previous version to this version and the upgrade worked as expected. I.e the customMetrics secret got moved to a new configmap.

Reply:
you are right, thx for clarification!

Reply:
at least https://github.com/kubernetes/helm/blob/master/docs/charts.md#charts-and-versioning does not provide a hint; but I think you are right here, as the `values.yaml` (API) does not change in a BC way